### PR TITLE
Core: prepare server for removal of names in multidata

### PR DESCRIPTION
**Pull request by Berserker66**
## What is this fixing or adding?
prepare server for removal of names in multidata

## How was this tested?
ran a generation and hosted it

### CODE REVIEW

**Comment by Berserker66** [comment]
I loaded a file not actually made by this branch on accident, turns out the server still reads these fields. So fixing that now.

File: Utils.py [hunk 1]
```diff
@@ -38,7 +38,7 @@ class Version(typing.NamedTuple):
     build: int
 
 
-__version__ = "0.3.8"
+__version__ = "0.3.9"
```
Comment (black-sliver) [inline Utils.py:41]: already?

**Review by black-sliver — COMMENTED** [review]
Do i understand correctly - this does partially what you wanted, and we'll have another PR after 0.3.9 that does the rest?

**Comment by Berserker66** [comment]
> Do i understand correctly - this does partially what you wanted, and we'll have another PR after 0.3.9 that does the rest?

yes. I thought we could do the whole thing already, but deprecation wasn't done correctly ahead of time.

**Review by black-sliver — APPROVED** [review]

**Pull request merged**